.travis.yml

```diff
@@ -29,7 +29,7 @@ compiler:
 
 env:
     - CONFIG_OPTS="" DESTDIR="_install" TESTS="-test_fuzz"
-    - CONFIG_OPTS="no-asm -Werror --debug no-afalgeng no-shared enable-crypto-mdebug enable-rc5 enable-md2 enable-aria" TESTS="-test_fuzz"
+    - CONFIG_OPTS="no-asm -Werror --debug no-afalgeng no-shared enable-crypto-mdebug enable-rc5 enable-md2 enable-aria enable-sm4 enable-sm4" TESTS="-test_fuzz"
```

Comment:
Twice?

Reply:
Thank you @paulidale ... not enough sleep! Addressed now.